CanSat-code/CanSat-container/lib/XBEE.cpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+#include <SoftwareSerial.h>
+#include <Wire.h>
+
+#define RX_pin 12
+#define TX_pin 13
+
+namespace CanSat
+{
+    SoftwareSerial receiverSerial(RX_pin, TX_pin);
+    String c = "";
+
+        void Initialize()
+        {
+            Serial.begin(9600);
```

Comment:
we shouldn't need to use serial in this case since we are only trying to have this class interface with the serial for the radio devices